cluster-operator/src/main/java/io/strimzi/operator/cluster/ClusterOperatorConfig.java

```diff
@@ -233,6 +242,16 @@ private static long parseTimeout(String timeoutEnvVar, long defaultTimeout) {
         return timeout;
     }
 
+    private static int parseInt(String envVar, int defaultValue) {
+        int defalt = defaultValue;
```

Comment:
minor typo "defalt"

Reply:
not a typo, since `default` is a reserved keyword.  I can change to something else if you prefer.

Reply:
right :-) yeah another name would be better then!